### PR TITLE
PLAT-136903: Migrate storybook from 5 to 6

### PR DESCRIPTION
**Pull request by jeongran1230**
This PR updates storybook 5.x to 6.

Enact-DCO-1.0-Signed-off-by: Jeongran Jang (jeongran.jang@lge.com)

### CODE REVIEW

File: addons/knobs/button.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+ * The config object has a `defaultProps` key which is an object with keys that correlate to the
+ * `name` argument.
+ *
+ * Arguments:
+ * * knob/property name string
+ * * Config object with at least a `defaultProps` key containing a map of props and their default values
+ * * (Optional) a sample-specific initially selected value
+ */
+
+const button = (name, callback, Config, preferredValue) => {
```
Comment (YunhoLee0) [inline addons/knobs/button.js:19]:
```suggestion
const button = (name, callback, Config) => {
```

File: addons/knobs/button.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+	if (typeof Config === 'string' || Config == null) {
+		// Config wasn't set, or was omitted, causing the preferredValue to be the last value. Reassignment dipsy-doodle.
+		preferredValue = Config;
+		Config = {};
```
Comment (YunhoLee0) [inline addons/knobs/button.js:23]:
```suggestion
		const groupId = Config;
		Config = {};
		Config.groupId = groupId;
```

File: addons/knobs/button.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+	if (!Config.defaultProps) {
+		Config.defaultProps = {};
+	}
+
+	// If there's no group ID but there is a display name, use that for the group ID
+	if (Config.displayName && !Config.groupId) {
+		Config.groupId = Config.displayName;
+	}
+
+	return nullify(buttonKnob(name, callback, preferredValue || Config.defaultProps[name], Config.groupId));
```
Comment (YunhoLee0) [inline addons/knobs/button.js:36]:
```suggestion
	return nullify(buttonKnob(name, callback, Config.groupId));
```

Comment (MikyungKim) [reply addons/knobs/button.js]: 🆗

**Comment by YunhoLee0** [comment]
LGTM

**Review by YunhoLee0 — APPROVED** [review]

**Pull request merged**